### DISCUSSION TITLE
Save local file without asking anything

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#882] [Ideas]
### What feature would you like to see?

When I open a PDF using Zen Browser, make some changes on it and try to save it, the file would be saved as download. I need to choose where to save it and the system would ask me if I want to replace the original file. This process seems to be redundant. I hope local file be directly saved without asking me anything.

<img width="466" alt="image" src="https://github.com/user-attachments/assets/89c4df39-2fe6-464e-a235-036cbf442e44">

<img width="296" alt="image" src="https://github.com/user-attachments/assets/e1f63193-4bfb-486f-a774-c328ba7e1f87">

### What platform are you seeing the problem on?

macOS

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/207_

**Comment by SomeDudeBrosky** [comment]
That is dangerous for system integrity

**Comment by JSGRD22** [comment]
This would be a very helpful feature if implemented safely. I have a habit of spamming Ctrl+S and having to re-download local PDFs is a little annoying. Editing PDFs in Edge is smooth because it saves the updated file automatically.
